scss/adminLessonCard.module.scss

```diff
@@ -0,0 +1,43 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 2% 3% 2% 3%;
+  background: #ffffff;
```

Comment:
NIT: To stay consistent, I would import `colors.$bg-light` from `colors.module.scss`.